cdap-api/src/main/java/io/cdap/cdap/api/lineage/field/EndPoint.java

```diff
@@ -28,6 +28,7 @@
 public class EndPoint {
   private final String namespace;
   private final String name;
+  private transient String location;
```

Comment:
What is location? And why is it transient?

Reply:
Replied offline

Reply:
Offline discussion summary: Have location as a property instead of as a field of its own